config/routes.rb

```diff
@@ -6,4 +6,13 @@
       resources :merchants, only: [:index, :show]
     end
   end
+
+  namespace :api do
+    namespace :v1, defaults: {format: :json} do
+      namespace :invoices do
+        get "/find_all", to: "find#index"
+        get "/find", to: "find#show"
```

Comment:
little bitty nit-picky thing but we should aim to keep our indentations consistent :)